ignite/handlers/lr_finder.py

```diff
@@ -302,12 +305,12 @@ def lr_suggestion(self) -> Any:
         if not self._history:
             raise RuntimeError("learning rate finder didn't run yet so lr_suggestion can't be returned")
         loss = self._history["loss"]
-        min_loss_idx = torch.tensor(loss).argmin()
+        min_loss_idx = loss.clone().detach().argmin()
```

Comment:
@OBITORASU thanks for pointing out that. @KickItLikeShika I think we should make sure that `loss` is a float and not a tensor requires grad:
https://github.com/pytorch/ignite/blob/65d9844de3b7faf23371dd4c82cd16d02d7e67aa/ignite/handlers/lr_finder.py#L138-L140
In this way we can reuse `torch.tensor(loss).argmin()` without any need to detach.

Reply:
Doing a fallback on this one to test out the other changes, once that is done I will come back to fixing this one again.